recipes/odbc/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class OdbcConan(ConanFile):
+    name = 'odbc'
+    version = '2.3.7'
+    description = 'Package providing unixODBC or Microsoft ODBC'
+    url = 'https://github.com/conan-io/conan-center-index'
+
+    license = 'LGPL/GPL'
+
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    options = {'shared': [True, False], 'fPIC': [True, False]}
+    default_options = {'shared': False, 'fPIC': True}
+
+    _source_subfolder = 'source_subfolder'
+
+    def configure(self):
+        del self.settings.compiler.libcxx  # Pure C
+        del self.settings.compiler.cppstd
+        if self.settings.os == "Windows" and not self.options.shared:
```

Comment:
at line 36 it doesn't build anything, and on the package method it doesn't copy any artifact. So, it accepts shared library, but I can't see that library in the package.